openfisca_france/parameters/prestations_sociales/solidarite_insertion/minima_sociaux/rsa/rsa_cond/patrimoine/taux_interet_forfaitaire_epargne_non_imposable.yaml

```
@@ -1,5 +1,6 @@
 description: Taux d'intérêt forfaitaire appliqué à l'épargne générant des revenus non imposables
-unit: /1
 values:
   2009-01-01:
     value: 0.03
+metadata: 
+  unit: /1
```

De même, typo de retour à la ligne :
```suggestion
  unit: /1

```